src/InputField/index.js

```diff
@@ -198,14 +199,15 @@ export const Input = styled(
   color: inherit;
   background-color: transparent;
   cursor: inherit;
-  flex: 1 auto;
+  flex: 1 1 20%;
   width: 100%;
   height: 100%;
+  box-sizing: border-box;
   border-radius: ${({ theme }) => theme.orbit.borderRadiusNormal};
   z-index: 2;
 
   // FIREFOX flexbox bug: the input doesn't shrink properly
-  min-width: 0;
+  min-width: 100px;
```

Comment:
Bug #756 appears when `100px`. Is it really needed when you setting `flex-basis: 20%`?